### DISCUSSION TITLE
What is `@internationalized/date`'s equivalent of `date-fns`' `parseISO`?

### DISCUSSION THREAD

**Discussion by shyakadavis** [adobe/react-spectrum#7084] [Q&A]
Hello,

`date-fns` has a method called `parseISO` which parses the given string in ISO 8601 format and returns an instance of Date.

Was wondering if anyone could tell me the equivalent of said method in `@internationalized/date`.

Thank you.

**Comment by snowystinger** [comment]
I think you're looking for parseDateTime?

https://react-spectrum.adobe.com/internationalized/date/CalendarDateTime.html#introduction

**Reply by shyakadavis** [reply]
Hello, @snowystinger Thanks for taking the time.

I think you're correct, however, for my use-case, it's not working. (Please ignore the rest if it's out of scope of the lib)

So basically, I'm working with a charts lib called LayerChart, which, from what I understand, uses `scaleTime` (which is local time) when scaling/plotting dates on an x-axis for example.

And so, the recommendation was to use `date-fns`' `parseISO` to parse them as local time, where the input was something simple as `2024-01-01`, `2024-02-01`, `2024-03-01`...

Had a different requirement to only use `@internationalized/date` for date stuff, ergo the question.

**Reply by arzatskis** [reply]
@shyakadavis then you probably need `parseDate` https://react-spectrum.adobe.com/internationalized/date/CalendarDate.html
